packages/nimble-components/src/wafer-map/modules/matrixRenderer.ts

```diff
@@ -0,0 +1,23 @@
+import * as Comlink from 'comlink';
+import type { WaferMap } from '..';
+import { workerCode } from '../workers/renderWorker';
+
+/**
+ * Responsible for drawing the dies inside the wafer map, adding dieText and scaling the canvas
+ */
+export class MatrixRenderer {
+    public readonly workerOne: Comlink.Remote<Worker>;
+
+    public constructor(private readonly wafermap: WaferMap) {
+        const blob = new Blob([workerCode], { type: 'text/javascript' });
+        const url = URL.createObjectURL(blob);
+        this.workerOne = Comlink.wrap(new Worker(url));
+    }
+
+    public isWorkerHealthy(): boolean {
```

Comment:
maybe requesting a status from the worker using comlink is also a good idea

Reply:
I added a different health check logic, you can check it in render-worker